layers/meta-balena-allwinner/recipes-bsp/u-boot/u-boot_2018.03.bb

```diff
@@ -0,0 +1,42 @@
+DESCRIPTION="Upstream's U-boot configured for sunxi devices"
```

Comment:
What u-boot version is the BSP layer using now? Why can't we use that one too?

Reply:
It's 2018.09 but that is failing when meta-balena tries to apply balena-specific-env patches.

Reply:
Then those patches need to be redone for the new u-boot. I don't understand why we should pin u-boot to an older release rather than redo those patches.

Reply:
What is your proposal for this? Can those patches be done in meta-balena-allwiner?